pandas/_libs/internals.pyx

```diff
@@ -418,12 +416,10 @@ def get_blkno_indexers(int64_t[:] blknos, bint group=True):
             else:
                 tot_len = sum(stop - start for start, stop in slices)
                 result = np.empty(tot_len, dtype=np.int64)
-                res_view = result
 
                 i = 0
                 for start, stop in slices:
                     for diff in range(start, stop):
-                        res_view[i] = diff
```

Comment:
this is equivalent to setting `result[i] = diff`.  my guess is doing it like with the memoryview is intended for perf

Reply:
I see, the CI is not passing which is a clear sign that this is important.

Now all that I am wondering is how to get rid of that warnings during compilation.

Reply:
Yea the warnings here are really the big blocker for #32163 . If you find a solution that would be huge!